### ISSUE TITLE
17.4: Alt text for figures always displayed

### ISSUE THREAD

**Issue by RiversPharmD** [hadley/r4ds#1546]
It looks like the alt text for the two figures showing the impact of re-ordering factor levels is missing a wrapper/the wrapper is placed at the end of the captions?

![Screen Shot 2023-07-24 at 10 14 34 AM](https://github.com/hadley/r4ds/assets/40068731/3df5d369-f132-4aea-b59e-98df684b3fae)

**Comment by RiversPharmD** [comment]
This is also seen in 19.4:

![Screen Shot 2023-07-24 at 11 23 10 AM](https://github.com/hadley/r4ds/assets/40068731/27e5ab75-62a8-4da4-b369-4d257a0997e8)

**Comment by mine-cetinkaya-rundel** [comment]
Thank you!
